hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/hdfs/protocol/datatransfer/IOStreamPair.java

```diff
@@ -23,10 +23,12 @@
 import java.io.OutputStream;
 
 import org.apache.hadoop.classification.InterfaceAudience;
+import org.apache.hadoop.classification.InterfaceStability;
 import org.apache.hadoop.io.IOUtils;
 
 /**
  * A little struct class to wrap an InputStream and an OutputStream.
+ * Inputstream and OutputStream are closed on close call.
```

Comment:
1. maybe the history of this class should be described as in "moved from hdfs to hadoop-common so yarn can remove HDFS dependencies"
2. And add that package-info.java file, mark the whole package as InterfaceAudience.Private

Reply:
ack